pages/home.html

```diff
@@ -510,7 +513,7 @@ <h2 class="text-center">Sponsored by</h2>
 			game: '{{ project.title }}',
 				by: '{{ project.author }}',
 					url: '{{ project.url }}',
-						image: '{{ project.image }}',
+						image: '{{ project.gallery[0] }}',
```

Comment:
Was that intentional?

Reply:
Right now in this branch the `.image` and the `.gallery[0]` is the same image. This is more relevant on the other branch which will take a new cover image and use the first image from the gallery for the hero backgrounds. I'll move it to the other PR